 #ADAPTIVE-FILTER Parameters
 adaptive_filter:

  # Filter settings
  enableImu: true
  enableWheel: true
  enableLidar: true
  enableVisual: true
  filterFreq: "l"

  # input filter
  alpha_lidar: 0.99
  alpha_visual: 0.999

  # gains  
  lidarG: 75
  wheelG: 0.5
  visualG: 0.1
  imuG: 100

  # Wheel odometry covariance adaptive positive constants
  gamma_vx: 0.05
  gamma_omegaz: 0.01
  delta_vx: 0.0001
  delta_omegaz: 0.00001

  # Topic names
  imuTopic: "/imu"
  wheelTopic: "/odom"
  FastLIO2_OdometryTopic: "/Odometry"
  cameraTopic: "/odom"
  filterTopic: "/filter_odom"

  # type of adaptative function
  type_func: 0                        # 0: linear; 1: exponential; 3: Fuzzy with 2 states; 4: original message

  # type of cameras
  camera_type: 2                      # 1: tracking; 2: depth

